src/app/components/button/index.tsx

```diff
@@ -140,8 +142,9 @@ function ActionButton({
       ) : (
         <>
           {src && <ButtonImage src={src} />}
-          {icon && <ButtonIconContainer>{icon}</ButtonIconContainer>}
+          {icon && iconPosition === 'left' && <ButtonIconContainer>{icon}</ButtonIconContainer>}
           <ButtonText warning={warning}>{text}</ButtonText>
+          {icon && iconPosition === 'right' && <ButtonIconContainer>{icon}</ButtonIconContainer>}
```

Comment:
I added it because we now have it in our design, it's used for the tx review modal "Next" button that I implemented in this PR, here's the screenshot from Figma:
<img width="300" alt="Monosnap New Product library – Figma 2023-11-09 14-41-25" src="https://github.com/secretkeylabs/xverse-web-extension/assets/36603049/c1c73a18-8bb5-4124-b686-9574be249d34">